src/types.ts

```diff
@@ -14,7 +14,7 @@ export type Amount = BigNumber
 
 export enum ServiceType {
   DiscoveryProvider = 'discovery-provider',
-  CreatorNode = 'creator-node'
+  ContentNode = 'creator-node'
```

Comment:
this guy

Reply:
i believe this is the only instance of something hardcoded in the contracts

Reply:
Yeah, I don't think it's actually even hardcoded in the contracts. 
We just need to make a tx with the serviceTypeManager contract to register the content-node and de-register the creator-node, then re-register the services.
But, I asked roneil and he said it was fine just changing it in the UI for now.